src/openrct2/peep/Peep.h

```diff
@@ -511,6 +511,7 @@ enum PeepInvalidate
     PEEP_INVALIDATE_PEEP_2 = 1 << 2,
     PEEP_INVALIDATE_PEEP_INVENTORY = 1 << 3,
     PEEP_INVALIDATE_STAFF_STATS = 1 << 4,
+    PEEP_INVALIDATE_PEEP_ACTION = 1 << 5     // Currently set only when guest_heading_to_ride_id is changed
```

Comment:
```suggestion
    PEEP_INVALIDATE_PEEP_ACTION = 1 << 5, // Currently set only when guest_heading_to_ride_id is changed
```

Reply:
Whoops, turns out I should actually add that pre-commit hook instead of pasting the command and accidentally dropping the *.h part. This & contributors.md is in the second commit (feel free to squash both commits into one).

Reply:
You've accidentally left two spaces between the comma and slash, though. ;) As a result, the clang-format CI job still fails.

Reply:
Okay, this is now becoming embarrassing. The second space was a pre-commit hook test to make sure it works. And it worked! So I removed that second space again and committed, but forgot to "git add" the fixed file before committing. So I blame the pre-commit hook. :-)